### PR TITLE
[MOD-62][Feature] Install i18n and add translation file

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose
Add translation file.

## Changes
* Install i18n
* Add common words to translation file
* Define missing values in theme.js

### CODE REVIEW

File: app/routes/application.js [hunk 1]
```diff
@@ -10,4 +10,28 @@ import OSFAgnosticAuthRouteMixin from 'ember-osf/mixins/osf-agnostic-auth-route'
...
+        if (navigator.languages && navigator.languages.length) {
+            for (let lang of navigator.languages) {
+                if (availableLocales.includes(lang)) {
+                    locale = lang;
+                    break;
+                }
+            }
+        }
+        // Backup for Safari (uses system settings)
+        else if (navigator.language && availableLocales.includes(navigator.language)) {
```
Comment (chrisseto) [inline app/routes/application.js]: Our style guide prefers `} else if (...) {`. Putting the comment beneath the if shouldn't be too confusing

File: config/environment.js [hunk 2]
```diff
@@ -34,6 +34,9 @@ module.exports = function(environment) {
         },
         Reviews: {
             defaultProvider: 'osf',
+            i18n: {
```
Comment (chrisseto) [inline config/environment.js]: How does i18n know to read out of the `Reviews` key?

File: app/services/theme.js [hunk 3]
```diff
@@ -32,4 +32,17 @@ export default Ember.Service.extend({
         return `${config.OSF.url}register?${query}`;
     }),
 
+    // The provider object
+    provider: Ember.computed('id', function() {
```
Comment (chrisseto) [inline app/services/theme.js:36]: What do we need this for?

**Review by chrisseto — CHANGES_REQUESTED** [review]
2 questions and a nitpick

File: app/services/theme.js (hunk 3 above)
Comment (laurenbarker) [reply app/services/theme.js:36]: ember-osf errors if provider isn't defined :/

File: app/routes/application.js (hunk 1 above)
Comment (laurenbarker) [reply app/routes/application.js]: Sure. PSA: the code is directly copied from preprints.

File: config/environment.js (hunk 2 above)
Comment (laurenbarker) [reply config/environment.js]: Oops this addition shouldn't be here, it's already in the file.

**Comment by laurenbarker** [comment]
@chrisseto, changes addressed 💸

**Review by chrisseto — APPROVED** [review]

**Pull request merged**